modules/ROOT/pages/about-designing-a-mule-application.adoc

```diff
@@ -1,29 +1,25 @@
-= About Designing Mule Apps
+= Flow Designer
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :keywords:
 
-In Design Center, you create a Mule application in a project having one or more flows. A flow typically consists of one trigger and one or more connectors, components, or APIs.
+In Flow Designer, you create Mule applications as projects having one or more flows. A project contains the flows for one Mule application. A Mule application can have one or more flows. A flow consists of a sequence of _cards_, each card representing a connector, core Mule Runtime component, API, or module. Each card receives an input, carries out a specific task, and then sends the output of that task to the next card in the flow. The final card is a flow is usually one that sends a notification that the Mule application has completed its task or that sends data.
```

Comment:
I suggest  "with each card representing a connector,..."

"The final card in a flow..." instead of "The final card is a flow..."